src/index.html

```diff
@@ -9,6 +9,55 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="logo-img"
+          src="./images/logo.png"
+          alt="logo-icon"
+        >
+      </a>
+      <nav class="nav-list">
+        <ul class="list-header-menu ">
+          <li class="element-header-menu">
+            <a href="apple" class="is-active link">Apple</a>
+          </li>
```

Comment:
You need to change the class names to make them more informative. Here is an example on the screen. 
You need to fix it everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="is-active nav__link">Apple</a>
          </li>
```
<img width="1033" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/d73ab394-591e-421b-9d43-467034241ccb">